src/docs/product/dashboards/customize-dashboards/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Customize Dashboards
+sidebar_order:
+description: "Learn how to edit or create additional dashboards to suit your organization's needs."
+---
+
+<Note>
+
+Editing dashboards is only available if your organization is on either the Business plan or the Trial plan.
+
+</Note>
+
+When you visit Dashboards for the first time, you'll find a dashboard with a set of pre-built widgets. You can build a new dashboard or edit an existing dashboard by clicking on either the "Create Dashboard" or the "Edit Dashboard" buttons on the top right-hand corner.
```

Comment:
a thought: if either of these buttons move, the content won't be evergreen. If they're already pretty self-evident, we don't need to tell users to look in the top RH corner.

```suggestion
When you visit Dashboards for the first time, you'll find a dashboard with a set of pre-built widgets. You can build a new dashboard or edit an existing dashboard by clicking on either "Create Dashboard" or "Edit Dashboard" on the top right-hand corner.
```

Reply:
Based on the screenshots, the buttons are pretty self-evident, so I've implemented your suggestion in your suggested version.